nodeup/pkg/model/kubelet.go

```diff
@@ -35,6 +38,8 @@ type KubeletBuilder struct {
 
 var _ fi.ModelBuilder = &DockerBuilder{}
 
+const socatStaticBinaryURL = "https://github.com/aledbf/socat-static-binary/releases/download/v0.0.1/socat-linux-amd64"
```

Comment:
Might be worth making the URL configurable like [NODEUP_URL](https://github.com/kubernetes/kops/blob/master/upup/pkg/fi/cloudup/urls.go#L61) but personally I'm fine with it this way.